### PR TITLE
Add CNTK <-> Onnx tutorials

### PR DESCRIPTION
**Pull request by mx-iao**

### CODE REVIEW

**Comment by mx-iao** [comment]
Should we clarify that CreateModel() is a user-defined function?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 | --- | --- | --- | --- |
 | [Caffe2](http://caffe2.ai) | [onnx/onnx-caffe2](https://github.com/onnx/onnx-caffe2) | [Exporting](tutorials/Caffe2OnnxExport.ipynb) | [Importing](tutorials/OnnxCaffe2Import.ipynb) |
 | [PyTorch](http://pytorch.org/) | [part of pytorch package](http://pytorch.org/docs/master/onnx.html) | [Exporting](tutorials/PytorchOnnxExport.ipynb), [Extending support](tutorials/PytorchAddExportSupport.md) | coming soon |
-| [CNTK](https://github.com/Microsoft/CNTK) | coming soon | coming soon | coming soon |
+| [CNTK](https://github.com/Microsoft/CNTK) | coming soon | [Exporting](tutorials/CntkOnnxExport.ipynb) | [Importing](tutorials/OnnxCntkImport.ipynb) |
```
Comment (prasanthpul) [inline README.md]: The installation cell should say its part of CNTK and link to the CNTK installation instructions

**Comment by dzhulgakov** [comment]
Thanks a lot!

Since it's a tutorial, not documentation - can I kindly suggest to actually show case a model working end to end like other tutorials? That would make it more hands-on and also we can utilize the tutorials for continuous testing (we plan to do it for other ones as well)

**Comment by mx-iao** [comment]
We are planning on extending the tutorials to include examples using a pretrained model once we release CNTK v2.3 tomorrow, which will have stable support for ONNX. In the meantime we wanted to at least have some brief documentation up. We could add a line to the notebooks saying that an e2e example is coming soon, how does that sound?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**